src/common/base/src/utils/topic_util.rs

```diff
@@ -0,0 +1,55 @@
+use regex::Regex;
+const EXCLUSIVE_SUB_PREFIX: &str = "$exclusive";
+pub fn is_exclusive_sub(sub_path: String) -> bool {
+    sub_path.starts_with(EXCLUSIVE_SUB_PREFIX)
+}
+pub fn decode_exclusive_sub_path_to_topic_name(sub_path: String) -> String {
+    let mut str_slice: Vec<&str> = sub_path.split("/").collect();
+    str_slice.remove(0);
+    format!("/{}", str_slice.join("/"))
+}
+
+pub fn exclusive_sub_path_regex_match(sub_path1: String, sub_path2: String) -> bool {
+    let sub_topic_name1 = if is_exclusive_sub(sub_path1.clone()) {
```

Comment:
pub fn exclusive_sub_path_regex_match(sub_path1: String, sub_path2: String)
=>
pub fn exclusive_sub_path_regex_match(sub_path1: &str, sub_path2: &str)

Use of "&" in function arguments reduces the number of times "clone()" is used.

Reply:
okay, i'll make this change